tests/php/LeftAndMainTest.php

```diff
@@ -155,6 +155,11 @@ public function testCanView()
         $menuItems = $menuItems->column('Code');
         sort($menuItems);
 
+        // filter out any menus from other modules that may be installed
+        $menuItems = array_filter($menuItems, function ($item) {
+            return strpos($item, 'SilverStripe-Admin-') === 0;
+        });
```

Comment:
Why is this needed now when it wasn't before?

Reply:
No longer required I think, will remove